src/models/userHelpers.js

```diff
@@ -337,7 +336,9 @@ async function getUsersFromCsv(d2, file, csv, { maxUsers, orgUnitsField }) {
         const data = userRows.map((userRow, rowIndex) =>
             getPlainUserFromRow(userRow, modelValuesByField, rowIndex + 2)
         );
-        const users = data.map(o => o.user);
+        const users = data.map(o =>
+            o.user.disabled ? { ...o.user, disabled: o.user.disabled.toLowerCase() } : o.user
```

Comment:
Sorry, I hadn't notice before, but this processing for the field disabled could be in "getPlainUserFromRow", right?